docs/guide/cell-references.md

```diff
@@ -160,7 +160,38 @@ copied between different places.
   </tbody>
 </table>
 
-## Circular reference
+## Range references
+
+In HyperFormula, a range is a group of two or more adjacent cells.
+
+You can reference ranges in the following ways:
+
+| Range reference type | Example | Reversed examples       |
+| -------------------- | ------- | ----------------------- |
+| Cell addresses       | A1:B2   | A2:B1<br>B1:A2<br>B2:A1 |
+| Columns              | A:B     | B:A                     |
+| Rows                 | 1:2     | 2:1                     |
+
+You can reference ranges:
+- Through relative references (A1:B2)
+- Through absolute references (A$1:$B$2)
+- Across different sheets (=Sheet2!A1:B2)
```

Comment:
I'd mention that we do support ranges that spans across multiple sheets (`Sheet1!A1:Sheet2!B2`). Ref: #952

Reply:
Also:
> If you don't specify a sheet name for the second address, it's assumed that it is the same as in first address, so `Sheet2!A1:B2` is equivalent to `Sheet2!A1:Sheet2!B2`

Reply:
> I'd mention that we do support ranges that spans across multiple sheets 

I'm adding the `Sheet1!A1:Sheet2!B2` example, but I'd say it's **across two different sheets**, not **multiple sheets**, right? Because you can't do `Sheet1!A1:Sheet2!B2:Sheet3!C3`

Reply:
Range `Sheet1!A1:Sheet3!B2` contains cells from `Sheet1` and `Sheet3` but also from all the sheets between them (ID-wise, not name-wise!!!) so e.g. `Sheet2` assuming that 
```
getSheetId('`Sheet1') = 1
getSheetId('`Sheet2') = 2
getSheetId('`Sheet3') = 3
```

See the example with seperate sheets for each month (the syntax is a bit different but the idea is the same): https://www.ablebits.com/office-addins-blog/2015/12/09/excel-3d-reference-formula/